docs/static/resiliency.asciidoc

```diff
@@ -1,6 +1,8 @@
 [[resiliency]]
 == Data resiliency
```

Comment:
@robbavey What do you think about this heading? I've expanded the info to include (default) mem queue? Something like "Queues and data resiliency" maybe?  Other ideas are welcomed.